spec/ParseUser.spec.js

```diff
@@ -1352,7 +1367,7 @@ describe('Parse.User testing', () => {
         var b = JSON.parse(body);
         expect(b.results.length).toEqual(1);
         var user = b.results[0];
-        expect(Object.keys(user).length).toEqual(6);
+        expect(Object.keys(user).length).toEqual(7);
```

Comment:
Bumped to 7 as ACL property is set.
:warning: installationId is set there.
Having installationId mapped on the user can be highly problematic when a user has multiple devices. it should be removed.